src/framework/core/js/DataBinding.js

```diff
@@ -772,7 +772,7 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
     };
 
     fluid.parseModelRelay = function (that, mrrec, key) {
-        var parsedSource = mrrec.source ? fluid.parseValidModelReference(that, "modelRelay record member \"source\"", mrrec.source) :
+        var parsedSource = mrrec.source !== undefined ? fluid.parseValidModelReference(that, "modelRelay record member \"source\"", mrrec.source) :
```

Comment:
Shall `mrrec.source` be trimmed so it handles strings with leading or trailing spaces?

Reply:
It is not specified to do that. In general we don't support extraneous spaces where the system expects paths or IoC references.